src/main/resources/assets/cubicchunks/lang/en_US.lang

```diff
@@ -17,6 +17,13 @@ cubicchunks.gui.customcubic.ores_tab_title=Customize World Settings%nPage 2 of 3
 cubicchunks.gui.customcubic.advanced_tab_title=Customize World Settings%nPage 3 of 3%nAdvanced Settings
 
 #################################
+###        options            ###
+#################################
+
+cubicchunks.gui.verticalCubeLoadDistance=Vertical Distance:
+cubicchunks.gui.cubes=cubes
```

Comment:
These 2 can be combined into `Vertical Distance: %d cubes` and you would just use `I18n.format("cubicchunks.gui." + option.name(), option.getValue())`